### PR TITLE
Initial Github Action for CI

### PR DESCRIPTION
**Pull request by robbiet480**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  test:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Gems
+      run: bundle install
+
+    - name: Install Pods
+      run: pod install --repo-update
```
Comment (zacwest) [inline .github/workflows/ci.yml]: does this require a `bundle exec`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+on:
+  pull_request:
+
+env:
+  FASTLANE_SKIP_UPDATE_CHECK: true
+  FASTLANE_XCODE_LIST_TIMEOUT: 30
+  MATCH_KEYCHAIN_NAME: "gh_actions"
+
+jobs:
```
Comment (zacwest) [inline .github/workflows/ci.yml:11]: is there any way to cache the bundler and cocoapods dependencies?

Comment (robbiet480) [reply .github/workflows/ci.yml:11]: Already done, just wasn't pushed 🙂

File: .github/workflows/ci.yml (hunk 1 above)
Comment (robbiet480) [reply .github/workflows/ci.yml]: Can't hurt.

**Pull request merged**